src/main/java/com/rultor/agents/build/Response.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2009-2015, rultor.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the rultor.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.rultor.agents.build;
+
+/**
+ * Build service response.
+ *
+ * @author Eugene Bukhtin (maurezen@gmail.com)
+ * @version $Id$
+ * @since 1.0
+ */
+public class Response {
+
+    /**
+     * Message.
+     */
+    private final String message;
+    /**
+     * Build status.
+     */
+    private final Status status;
+
+    /**
+     * Constructor.
+     * @param msg Message.
+     * @param stts Build status.
+     */
+    public Response(final String msg, final Status stts) {
+        this.message = msg;
+        this.status = stts;
+    }
+
+    /**
+     * Message getter.
+     * @return Message.
+     */
+    public final String getMessage() {
```

Comment:
Is the `get*` naming convention required by AppVeyor? Otherwise, let's avoid it. SImpler to name the method `message()`.

See http://www.yegor256.com/2014/09/16/getters-and-setters-are-evil.html